### PR TITLE
add cnspec docs on scanning arista

### PR DESCRIPTION
**Pull request by misterpantz**
#### Description

Add cnspec docs on scanning Arista network devices

#### Related issue

none

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [ ] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: docs/cnspec/cnspec-oper/arista.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Assess Arista EOS Security with cnspec
+sidebar_label: Arista
+displayed_sidebar: cnspec
+sidebar_position: 1
+description: Assess the security of your Arista EOS network devices using Mondoo's cnspec client
+image: /img/featured_img/mondoo-feature.jpg
+---
+
+cnspec assesses your Arista EOS network devices for misconfigurations that put your organization at risk. You can evaluate an Arista asset for compliance with security policies created by Mondoo or the community, or [create your own policies](/cnspec/cnspec-policies/write/). You also can write individual tests to run on the fly or include in automated tasks.
```
Comment (tas50) [inline docs/cnspec/cnspec-oper/arista.mdx:10]: We don't actually have our own policy for these at the moment

File: docs/cnspec/cnspec-oper/arista.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+For a list of Arista EOS resources you can test, read [Arista EOS Resource Pack Reference](/mql/resources/arista-pack/), [Mondoo Operating Systems (OS) Resource Pack Reference](/mql/resources/os-pack/) and [Mondoo Core Resource Pack Reference](/mql/resources/core-pack/).
+
+## Requirements
+
+To test an Arista EOS asset with cnspec, you must have:
+
+- [cnspec installed on your workstation](/cnspec/)
+- Access to the asset
```
Comment (tas50) [inline docs/cnspec/cnspec-oper/arista.mdx:19]: We should probably mention that for scanning we require access over their HTTPS API

Comment (misterpantz) [reply docs/cnspec/cnspec-oper/arista.mdx:19]: So "- Access to the network's HTTPS API"?

Comment (tas50) [reply docs/cnspec/cnspec-oper/arista.mdx:19]: Access to the device using the HTTPS API

File: docs/cnspec/cnspec-oper/arista.mdx (hunk 1 above)
Comment (tas50) [reply docs/cnspec/cnspec-oper/arista.mdx:10]: You could document that in addition, but we can still mention community written or custom policies here. We just don't have anything they can use out of the box

Comment (misterpantz) [reply docs/cnspec/cnspec-oper/arista.mdx:10]: Okay I do provide links to all the other commands.

**Review by tas50 — APPROVED** [review]

**Pull request merged**